stream/stream_dvb.c

```diff
@@ -1054,6 +1058,34 @@ static int dvb_streaming_start(stream_t *stream, char *progname)
     return 1;
 }
 
+void dvb_update_config(stream_t *stream)
+{
+    static int last_check = 0;
+    int now = (int)(mp_time_sec()*1000);
+
+    // Throttle the check to at maximum once every 0.1 s.
+    if (now != last_check) {
```

Comment:
The m_config_cache_update call should be very cheap if nothing changes, so throttling seems slightly questionable. But could still make sense, maybe to reduce the number of reinits if multiple options are changed at once. (Though you could still say it should react after a timeout after no further changes.)

Reply:
My main fear was that polling too often would indeed slow down `streaming_read`. I put a `printf` there initially, and especially shortly after tuning, it was called with really high rate. I think I'll keep the current throttling for now, a better idea would indeed be to wait until no further changes come, but until then probably the current solution is good enough.

Reply:
The function only does an atomic read (with no contention), which is probably still faster than querying the time and all that stuff, but yeah, good enough for now.